packages/gafl-webapp-service/src/pages/buy-or-renew/update-transaction.js

```diff
@@ -0,0 +1,9 @@
+import { BUY_OR_RENEW } from '../../uri.js'
+
+export default async request => {
+  const { payload } = await request.cache().helpers.page.getCurrentPermission(BUY_OR_RENEW.page)
+  const buyNewLicence = payload['buy-or-renew'] === 'buy-licence'
+  const permission = await request.cache().helpers.transaction.getCurrentPermission()
```

Comment:
This should be `request.cache().helpers.status.getCurrentPermission()`. This could get messy if we copy over what we have from the transaction cache into the status cache